include/arch/x86_64/arch_common.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _ARCH_X86_64_H
+#define _ARCH_X86_64_H
+
+/** Intel endbr64 instruction optcode.  */
+#define INSN_ENDBR64 0xf3, 0x0f, 0x1e, 0xfa
```

Comment:
That is arch-dependent. There is a version of this header for each architecture. Furthermore, this macro is used in both `libpulp` and `ulp`, so unless this is defined in this header it would have to be copy-n-pasted everywhere.